### PR TITLE
Added static CMS page.

### PR DESCRIPTION
**Pull request by tan-90**
Also addressed change requests from previous PRs.

### CODE REVIEW

File: src/components/widgets/SubHeader/index.tsx [hunk 1]
```diff
@@ -1,17 +1,74 @@
 import React from 'react'
+import classNames from 'classnames'
 
 import styles from './SubHeader.module.scss'
 
+type SubHeaderTextSize = 'small' | 'normal' | 'medium' | 'large'
```
Comment (tan-90) [inline src/components/widgets/SubHeader/index.tsx:6]: Maybe these should be on a common file? We should probably discuss if we're gonna end up using this in other places.

File: src/components/widgets/InlineCta/InlineCta.module.scss [hunk 2]
```diff
@@ -23,33 +16,21 @@
     background-image: url('../../../img/bg-full.png');
```
Comment (Lisenish) [inline src/components/widgets/InlineCta/InlineCta.module.scss:16]: After we merge tech branch we need to replace it to the new approach (please leave TODO)

File: src/components/widgets/Hero/index.tsx [hunk 3]
```diff
@@ -2,21 +2,34 @@ import React from 'react'
...
+import classNames from 'classnames'
 
 import styles from './Hero.module.scss'
 
 export interface HeroProps {
   title: string
   description: string
   ctaText: string
+  ctaHref: string
+  className?: string | string[]
```
Comment (Lisenish) [inline src/components/widgets/Hero/index.tsx:15]: Ok, just let's fix it everywhere

File: src/components/widgets/SubHeader/index.tsx (hunk 1 above)
Comment (Lisenish) [reply src/components/widgets/SubHeader/index.tsx:6]: While we don't use it in other places it's fine to have it here, let's keep local related things close

File: src/helpers/content.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export const baseUrl = 'https://fingerprintjs.com'
```
Comment (Lisenish) [inline src/helpers/content.ts:1]: It should be in `constants/` folder

File: src/templates/account-sharing-page.tsx [hunk 5]
```diff
@@ -9,6 +9,7 @@ import { PreviewTemplateComponentProps } from 'netlify-cms-core'
 import { ArrayElement } from '../helpers/types'
```
Comment (Lisenish) [inline src/templates/account-sharing-page.tsx:9]: We need to delete this

File: content/pages/cms-static-page.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (Lisenish) [inline content/pages/cms-static-page.md:1]: Need to change the name but we're waiting for Savannah's input for that

File: static/admin/config.yml [hunk 7]
```diff
@@ -105,3 +139,81 @@ collections:
                   { label: Button href, name: buttonHref, widget: string },
                 ],
             }
+      - file: 'content/pages/cms-static-page.md'
+        label: 'CMS Static Page'
```
Comment (Lisenish) [inline static/admin/config.yml:143]: Need to negotiate name with Savannah

File: src/templates/cms-static-page.tsx [hunk 8]
```diff
@@ -0,0 +1,267 @@
...
+          ctaHref
+        }
+        cardSection {
+          title
+          subtitle
+          cards {
+            icon {
+              childImageSharp {
+                fixed(width: 28, quality: 100) {
+                  ...GatsbyImageSharpFixed
```
Comment (Lisenish) [inline src/templates/cms-static-page.tsx:80]: Let's try to use `GatsbyImageSharpFixed_withWebp `it should have better performance on Google Chrome, but also test in different browsers after that

Comment (Lisenish) [reply src/templates/cms-static-page.tsx:80]: And same for the image below

**Review by Lisenish — CHANGES_REQUESTED** [review]

**Comment by Lisenish** [comment]
Good job, only minor things, let's fix them on Monday and deploy after we get final content

**Comment by Lisenish** [comment]
Do you know why it's empty on the preview?
https://deploy-preview-47--fingerprintjscom.netlify.app/pages/cms-static-page/

**Review by Lisenish — APPROVED** [review]
LGTM, going to merge it to feature branch now

**Pull request merged**